tests/integration/pilot/outboundtrafficpolicy/helper.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package outboundtrafficpolicy
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"istio.io/istio/pilot/pkg/model"
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/apps"
+	"istio.io/istio/pkg/test/framework/components/galley"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+	"istio.io/istio/pkg/test/framework/components/pilot"
+	"istio.io/istio/pkg/test/util/retry"
+)
+
+const (
+	IstioSystemScope = `
+apiVersion: networking.istio.io/v1alpha3
+kind: Sidecar
+metadata:
+ name: restrict-to-istio-system
+spec:
+ egress:
+ - hosts:
+   - "istio-system/*"
+`
+)
+
+// This test checks external traffic calls behave as expected. Rather than call real external endpoints,
+// we will restrict the config scope with a Sidecar resource, such that the client doesn't know
+// about the destination app, allowing it to follow the same path as a standard external call.
+func ExternalRequest(expectedCodes []string, t *testing.T) {
```

Comment:
Minor Nit: may I recommend naming this with some "Test" suffix/prefix somewehre? (e.g. RunExternalRequestTest)
Also, consider making t as the first parameter. I've tried to make all the framework methods that I touched to follow that convention already.